### PR TITLE
External order of G-vectors

### PR DESCRIPTION
**Pull request by toxa81**
Provide a possibility to use the host code's order of G-vectors. This is needed to get rid of wave-function reshuffling between QE and SIRIUS.

### CODE REVIEW

**Comment by toxa81** [comment]
bors try

**Comment by toxa81** [comment]
bors try

**Comment by toxa81** [comment]
bors try

**Comment by toxa81** [comment]
bors try

**Comment by toxa81** [comment]
bors try

**Comment by toxa81** [comment]
bors try-

**Comment by toxa81** [comment]
bors try

**Comment by toxa81** [comment]
bors try

**Comment by toxa81** [comment]
bors try

**Comment by toxa81** [comment]
bors try-

**Comment by haampie** [comment]
bors retry

**Comment by haampie** [comment]
bors try

**Review by mtaillefumier — APPROVED** [review]

**Review by simonpintarelli — APPROVED** [review]
LGTM.

File: src/SDDK/gvec.cpp [hunk 1]
```diff
@@ -459,28 +480,39 @@ void Gvec::init(FFT3D_grid const& fft_grid)
         }
     }
     // TODO: add a check for gvec_base (there is already a test for this).
+    init_gvec_local();
+    init_gvec_cart_local();
 }
 
 Gvec& Gvec::operator=(Gvec&& src__)
```
[the remark this answers is not in the document]
Comment (toxa81) [reply src/SDDK/gvec.cpp]: A good point! This should be fixed now.

**Comment by toxa81** [comment]
bors try

**Pull request merged**